ci/docker/runtime_functions.sh

```diff
@@ -784,7 +784,25 @@ build_docs() {
 #Runs Apache RAT Check on MXNet Source for License Headers
 nightly_test_rat_check() {
     set -ex
-    ./tests/nightly/apache_rat_license_check/license_check.sh
+    pushd .
+    
+    cd /work/deps/trunk/apache-rat/target
+
+    #Command has been run twice, once for the logs and once to store in the variable to parse.
```

Comment:
Not sure what this means.  Which logs are we referring to? Which variables?

Reply:
Yeah right, we could actually just run it and store the output in a variable as well as print it to the log at the same time.